datasets/GM0340_rhenen/GM0340_rhenen.derived.ad

```diff
@@ -32,31 +32,31 @@
 - households_insulation_employment_constant = 0.014
 - co2_emission_1990_aviation_bunkers = 4.6046
 - co2_emission_1990_marine_bunkers = 34.95619
-- co2_percentage_free = 0.4
-- co2_price = 0.02483
-- offshore_ccs_potential_mt_per_year = 65.0
+- co2_percentage_free = 0.85
+- co2_price = 0.0078
+- offshore_ccs_potential_mt_per_year = 41.95
```

Comment:
This seems like a generic update that applies regardless of the analysis_year. Are you sure that it should be reverted?